src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Alerts can be created by users with admin permission or higher from **Project Settings > Alerts > New Alert**.
+
+There are two types of alerts: Issue Alerts and Metric Alerts. 
+
+## Metric Alerts
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. They can be used for high-level monitoring of patterns, or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors, or volume of database errors.
```

Comment:
Remove passive. Suggest "Use metric alerts for high-level monitory of patters or fine-grained... Using metric alerts helps express lightweight ....

And remove the comma before ", or volume"